sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -201,8 +226,13 @@ public WebhooksService webhooks() {
   }
 
   @Override
-  public ProjectAnalysisService projectAnalysis() {
-    return projectAnalysisService;
+  public ProjectAnalysisService projectAnalysisOld() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ProjectAnalysisService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)